workflow-templates/gradle-check/action.yml

```
@@ -0,0 +1,47 @@
+name: Gradle check
+description: Run Gradle check
+
+inputs:
+  jdk-versions:
+    description: Comma-separated list of JDK versions and distributions
```

List the supported ones as I think of now:

```suggestion
    description: "Comma-separated list of JDK versions and distributions (supported ones: \"8-zulu\", \"8-temurin\", \"11-temurin\", and \"17-temurin\")"
```

I am not sure if the syntax is correct. Help me check please.